### PR TITLE
[ALLUXIO-2279]add final to concurrenthashset.java

### PR DESCRIPTION
**Pull request by lijun16**
MG1633044 https://alluxio.atlassian.net/browse/ALLUXIO-2279
add final to concurrenthashset.java

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: core/common/src/main/java/alluxio/collections/ConcurrentHashSet.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public boolean retainAll(Collection<?> c) {
 
   @Override
   public Object[] toArray() {
-    return mMap.keySet().toArray();
+    return mMap.keySet().toArray(); ConcurrentHashSet.java
```
Comment (gpang) [inline core/common/src/main/java/alluxio/collections/ConcurrentHashSet.java]: I think this line was modified unintentionally?

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**